core/tests/puppeteer-acceptance-tests/utilities/user/super-admin.ts

```diff
@@ -99,6 +176,203 @@ export class SuperAdmin extends BaseUser {
     showMessage(`User ${username} does not have the ${role} role!`);
     await this.goto(currentPageUrl);
   }
+
+  /**
+   * Unassigns a role from a user.
+   * @param {string} username - The username of the user.
+   */
+  async unassignRoleFromUser(username: string, role: string): Promise<void> {
+    role = role.replace(/ /g, '-');
+    await this.goto(AdminPageRolesTab);
+    await this.page.waitForSelector(roleEditorInputField);
+    await this.type(roleEditorInputField, username);
+    await this.clickOn(roleEditorButtonSelector);
+    await this.page.waitForSelector('div.justify-content-between');
```

Comment:
Please always store these as a constant... Ditto for the rest.

Reply:
updated.